docs/build.sh

```diff
@@ -24,7 +24,7 @@ arch=$(go env GOARCH)
 MDBOOK_VERSION="0.4.15"
 
 # translate arch to rust's conventions (if we can)
-if [[ ${arch} == "amd64" ]]; then
+if [[ ${arch} == "amd64" ]] || [[ ${arch} == "arm64" ]]; then
     arch="x86_64"
```

Comment:
This seems weird to me: why are you forcing `x86_64` on `arm64`? That should be mapped to `aarch64`.

Reply:
nothing to map to on the other side: https://github.com/rust-lang/mdBook/releases `x86_64` are the only asset types.

Reply:
I meant to add a "why" comment. I'll do that.

Reply:
Then don't include `arm64` as it won't run on that platform.

Reply:
If this is to support macOS, you need to isolate this to only Darwin builds.

Reply:
> Then don't include arm64 as it won't run on that platform.

This is to support M1.

Reply:
> If this is to support macOS, you need to isolate this to only Darwin builds.

Will do